### DISCUSSION TITLE
Show progress of Cargo Crates and Lost Packs?

### DISCUSSION THREAD

**Discussion by Ursling** [spicyboys/drg-completionist#139] [Ideas]
Title.

**Comment by rob0rt** [comment]
This was something we had back in v4 and a feature I'd love to have again. It's perfectly suited for our new "overview" page (maybe as a new tab?). 

When copying the data over to v5 I didn't preserve the source for every item so that'd need to be backfilled, but this is totally doable with just a bit of work.

**Reply by Ursling** [reply]
Awesome! I hope to see it in the future!
